### DISCUSSION TITLE
Request for Assistance with Sim2Real Transformation Using Emitter Receiver in DeepBots

### DISCUSSION THREAD

**Discussion by abdul-mannan-khan** [bitcraze/discussions#1102] [Q&A]
Hello Bitcraze,

Thank you for sharing this amazing support repository for Crazyflie.
I am wondering about Sim2Real transformations, specifically within the context of using emitter-receiver configurations in [DeepBots](https://github.com/aidudezzz/deepbots). I am seeking guidance or any helpful resources that could steer me in the right direction for this endeavor.

My current project involves a Sim2Real transformation where I am attempting to bridge the gap between the simulated environment and the real-world application using a setup based on [DeepBots](https://github.com/aidudezzz/deepbots). I am particularly focused on understanding and implementing the emitter-receiver functionality effectively for this transition.

As a beginner in this area, any advice, tutorials, or shared experiences would be incredibly valuable to me. I am especially interested in any best practices or common pitfalls to avoid during this process.

Thank you in advance for your time and assistance. Your insights will be greatly appreciated and will surely contribute significantly to my learning journey in this exciting field.

**Comment by knmcguire** [comment]
Hi! and sorry for the late reply. It has been the winter holidays here so many of us were on holidays.

We don't have experience with deepbots ourselves but soudns interesting! From our side, we can perhaps help more with the connection to the crazyflie than the webots-deepbots connection.

Here are some docs to get started:
- Webots tutorial: https://www.bitcraze.io/documentation/tutorials/getting-started-with-simulation/
- Stepbystep guides to use the crazyflie python library: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/
- The ROS2 variant to control the crazyflie: https://imrclab.github.io/crazyswarm2/
- A lecture of 101 info about the crazyflie; https://youtu.be/es69Nf0Wlwc

Talking about most common pittfalls that happens a lot here in the forum, is immediately jumping into a large project and trying to get everything to work right away. I see people wanting to fly 10 crazyflies autonomously right from the bat with aidecks without ever having flown one before. Start small, make sure you have 1 crazyflie that can take off with the crazyflie python library and start from there.
